lib/Modules/HyperVCIHooks/HyperVCIHooks.psm1

```diff
@@ -785,7 +785,7 @@ function Initialize-GitRepositories {
         [ValidateSet("hyperv", "ovs")]
         [string]$NetworkType,
         [string]$BranchName,
-        [ValidateSet("openstack/nova", "openstack/neutron", "openstack/networking-hyperv", "openstack/quantum", "openstack/compute-hyperv")]
+        [ValidateSet("openstack/nova", "openstack/neutron", "openstack/networking-hyperv", "openstack/quantum", "openstack/compute-hyperv", "openstack/cinder", "openstack/none")]
```

Comment:
there is no openstack/quantum in a long time, afaik. Also, what's openstack/none?